pylint_odoo/checkers/no_modules.py

```diff
@@ -327,10 +348,29 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    def open(self):
+        super(NoModuleChecker, self).open()
+        self.config.deprecated_field_parameters = \
+            self.colon_list_to_dict(self.config.deprecated_field_parameters)
+
+    def colon_list_to_dict(self, colon_list=None):
+        """ \
+        Converts a colon list to a dictionary
+        Example:
+            >> self.colon_list_to_dist(['colon:list', 'just_key'])
+            {'colon': 'list', 'just_key': None}
+        """
+        tmp_dict = {}
+        for item in colon_list if isinstance(colon_list, list) else []:
+            prop = item.split(":")
+            tmp_dict[prop[0]] = prop[1] if len(prop) > 1 else None
```

Comment:
This would fail if the key doesn't have a value. Like this:
![screenshot-repl it-2016-12-26-09-20-59](https://cloud.githubusercontent.com/assets/8657959/21482607/d910a3d6-cb4c-11e6-97f5-9cfdd7c25afb.png)
So what do you think if we also change the string for the keys that won't have replacements to end with a colon, and have an empty right side: `['key1:val1', 'key2:val2', 'key3:']`? It will evaluate as a key with an empty value (which is nice because is a falsy value).
![screenshot-repl it-2016-12-26-09-34-47](https://cloud.githubusercontent.com/assets/8657959/21482765/93463986-cb4e-11e6-82b0-1e86fc89437c.png)